File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        map<int, int> mpp;
        for (auto i : nums) {
            mpp[i]++;
        }
        int cnt = 0;
        for (auto it : mpp) {
            nums[cnt] = it.first;
            cnt++;
        }
        return cnt;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = len(nums)
        a = []
        a.append(nums[0])
        for i in range(1, n):
            if nums[i] != nums[i - 1]:
                a.append(nums[i])

        for i in range(len(a)):
            nums[i] = a[i]
        return len(a)
